dataclients/kubernetes/kube.go

```diff
@@ -349,14 +349,25 @@ func (c *Client) convertDefaultBackend(i *ingressItem) (*eskip.Route, bool, erro
 	return r, true, nil
 }
 
-func (c *Client) convertPathRule(ns, name, host string, prule *pathRule) (*eskip.Route, error) {
+func (c *Client) convertPathRule(ns, name, host string, prule *pathRule, servicesURLs map[string]string) (*eskip.Route, error) {
 	if prule.Backend == nil {
 		return nil, fmt.Errorf("invalid path rule, missing backend in: %s/%s/%s", ns, name, host)
 	}
-
-	address, err := c.getServiceURL(ns, prule.Backend.ServiceName, prule.Backend.ServicePort)
-	if err != nil {
-		return nil, err
+	serviceKey := ns + prule.Backend.ServiceName + prule.Backend.ServicePort.String()
```

Comment:
Why that can reduce conflicts? With the same values, it will generate the same key. I mean, if this key:
`namespaceservice80`
have a conflict, the key:
`__srv__namespace__service__80`
will hace the exact same probability of collisions.